subset/switches/src/main/java/switchtest/Main.java

```diff
@@ -14,9 +18,28 @@ public static void main(String[] args) throws Exception {
 
     boolean supportsPOE = args[2].equals("true");
 
-    SwitchInterrogator switchInterrogator = new SwitchInterrogator(ipAddress, interfacePort, supportsPOE);
+    SupportedSwitchModelsEnum switchModel = null;
+    try {
+      switchModel = SupportedSwitchModelsEnum.valueOf(args[3]);
+    } catch (Exception e) {
+      System.out.println(
```

Comment:
no favorites... either success or hard error.

Reply:
Same note as above

Reply:
This should be resolved now